util/cosmos/iscn/sign.js

```diff
@@ -135,8 +135,12 @@ export function preformatISCNPayload(payload) {
     recordNotes = '',
   } = payload;
 
-  let stakeholders = inputStakeholders || [];
+  let stakeholders = inputStakeholders ? [...inputStakeholders] : [];
```

Comment:
If default `inputStakeholders` is `[]`, would `stakeholders` fallback here?

Reply:
spreading [] is still [] so it's ok, but spreading undefined is error

Reply:
Since `!!inputStakeholders`  will always be `true`, will `let stakeholders = [...inputStakeholders];` works the same way?

Reply:
The original code is `  let stakeholders = inputStakeholders || [];`
So im not sure if input can be null or false despite having default value